openlibrary/catalog/get_ia.py

```diff
@@ -78,7 +79,7 @@ def get_marc_record_from_ia(identifier):
 
     # If that fails, try marc.bin
     if marc_bin_filename in filenames:
-        data = urlopen_keep_trying(item_base + marc_bin_filename).read()
+        data = urlopen_keep_trying(item_base + marc_bin_filename).text
```

Comment:
@hornc Can you please review these changes?  My sense is we want `data` to be `bytes`, not `str` here.  i.e. `requests.get().content`, not `requests.get().text`.

Reply:
Looks like MarcBinary needs bytes:
```python
class MarcBinary(MarcBase):
    def __init__(self, data):
    # def __init__(self, data: bytes) -> None:  # Python 3 type hint
```